tests/Console/Command/FixCommandTest.php

```diff
@@ -52,7 +52,7 @@ public function testEmptyRulesValue()
      */
     public function testEmptyFormatValue()
     {
-        $cmdTester = $this->doTestExecute(array('--using-cache' => 'not today'));
+        $cmdTester = $this->doTestExecute(array('--using-cache' => 'not today', '--rules=switch_case_semicolon_to_colon'));
```

Comment:
btw, why passing `--rules` fixing the issue?

Reply:
the test check the result of fixing this file, which might have diff. with the default set on diff. PHP versions

Reply:
well, that change didn't fix travis.